apps/communications/contacts/js/views/form.js

```diff
@@ -822,6 +854,12 @@ contacts.Form = (function() {
       formHeader.addEventListener('action', cancelHandler);
       doMatch(contact, callbacks);
     });
+    var title = formTitle.getAttribute('data-l10n-id');
+    if (title === 'editContact') {
+      Contacts.showStatus({
+        id: 'updateContact'
+      });
+    }
```

Comment:
actually the form to add the contact details and edit details are almost same and the only way they are differentiating is by changing the data-l10n-id. ie i used it.